src/main/java/com/google/sps/servlets/DestinationDataServlet.java

```diff
@@ -44,10 +46,11 @@
   private static final String HINT3_PARAMETER = "hint3";
   private static final String OBSCURITY_PARAMETER = "obscurity";
   private static final String TAG_PARAMETER = "tag";
-  private static final String REDIRECT_URL = "/destination-data";
+  private static final String REDIRECT_URL = "/index.html";
 
-  // Temporarily stores the destination created by the user
-  public List<Destination> destinations = new ArrayList<>();
```

Comment:
If this is storing the destination that has just been created by the user, shouldn't this be just a Destination object instead of an array? Or if you're adding all the created Destinations to this array for now, maybe update this comment so it's more clear that it's all Destinations instead of just one

Reply:
I was using that to store the destinations created during the prototype. I removed it because now we are storing in datastore.